### PR TITLE
docs: use new `nuxi module add` command in installation

### PR DESCRIPTION
**Pull request by danielroe**
## Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Description

This updates the documentation to use the [`nuxi module add` command](https://github.com/nuxt/cli/pull/197) which should simplify docs a bit and also improve user experience as there's no need to add to `nuxt.config` manually.

It's documented [here](https://nuxt.com/docs/api/commands/module#nuxi-module-add).

I may have missed a few spots in the documentation as I'm doing this across the modules ecosystem assisted by the power of regular expressions ✨, so I'd appreciate a review 🙏

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes (if not applicable, please state why)

### CODE REVIEW

**Comment by flozero** [comment]
I will have a look next weekend thank you @danielroe  🙏🏼

**Comment by fuentesloic** [comment]
> I will have a look next weekend thank you @danielroe 🙏🏼

cc @flozero will do to! 🚀

File: README.md [hunk 1]
```diff
@@ -61,14 +61,7 @@ const stripe = await useClientStripe()
 1. Add `@unlok-co/nuxt-stripe` dependency to your project
```
Comment (fuentesloic) [inline README.md:61]: @danielroe I would add more detail on what is going to happen in my project. wdyt? I'm doing the same on leaflet project as I was playing with. Will ping you there as well.
```suggestion
Run `npx` command to add `@unlok-co/nuxt-stripe` into your `package.json` and `nuxt.config.ts`
```

File: README.md [hunk 2]
```diff
@@ -61,14 +61,7 @@ const stripe = await useClientStripe()
...
-
-# Using yarn
-yarn add --dev @unlok-co/nuxt-stripe
-
-# Using npm
-npm install --save-dev @unlok-co/nuxt-stripe
+npx nuxi@latest module add stripe-next
 ```
 
 2. Add `@unlok-co/nuxt-stripe` to the `modules` section of `nuxt.config.ts`
```
Comment (fuentesloic) [inline README.md:67]: We can remove this as it will be removed when running `npx nuxi@latest module add stripe-next`

Comment (fuentesloic) [inline README.md:67]: We could remove this as it will be automatically added by `npx nuxi@latest module add stripe-next`

```suggestion
```

**Comment by fuentesloic** [comment]
@danielroe @flozero good to me, it's working on my side. A few notes, feel free to adapt or decline! 👍

**Review by fuentesloic — APPROVED** [review]

**Pull request merged**